### PR TITLE
Upgrade to Celery 4.0.2

### PR DESCRIPTION
**Pull request by comzeradd**
This would need a rebase after #1340 is merged. Especially for the `djecelery` parts that should be removed from this commit.

### CODE REVIEW

File: remo/settings/base.py [hunk 1]
```diff
@@ -300,10 +298,6 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 # Paginator settings
 ITEMS_PER_PAGE = 20
 
-# django-celery setup
-import djcelery  # noqa
-djcelery.setup_loader()
```
Comment (akatsoulas) [inline remo/settings/base.py]: Shouldn't all the `djcelery` related entries be part of the update to `3.1.25` version?

File: requirements/prod.txt [hunk 2]
```diff
@@ -11,19 +11,20 @@ MarkupSafe==0.23 \
...
-    --hash=sha256:6441e798f8863744c90574de253bed328c477fd2c86caf52b8d9d85f1ecb9f96 \
-    --hash=sha256:4d64ed1b9e0e73095f5cfa87f0e97ddb4c840049e8efeb7e63b46118ba1d623a \
-    --hash=sha256:39ef0541e489b1cf5515cedf1b1f99de585fd660463cdcde43a03d7d92042782 \
-    --hash=sha256:685b1994026572177458af143cc24a2082f400f18e63fdc8e2ed68dd93bb0f49 \
-    --hash=sha256:8446013dfbde9b3f4f9bfea09418ddd526760a8798c41badd3f707a98ad0745b \
-    --hash=sha256:d5ff706089e9150760d238b66203a24d4a0a6175160410191995ff04d1893faa \
-    --hash=sha256:a78b484d5472dd8c688f8b3eee18646a25c66ce45b2c26652850f6af9ce52b17 \
-    --hash=sha256:c4ee70cb407f9284517ac368f121cf0796a7134b961e53d9daf1aaae8f44fb90 \
-    --hash=sha256:00f35fb0e7e2fc348cd091d25d36c6768b721463d12396d68a9491a8a5f91ecf
+pytz==2016.10 \
```
Comment (akatsoulas) [inline requirements/prod.txt]: Why is this part of this PR?

File: remo/settings/base.py (hunk 1 above)
Comment (comzeradd) [inline remo/settings/base.py]: Yes, these will be removed from the commit once it's rebased (see initial comment). If I leave the djcelery parts, celery4 won't work.

File: requirements/prod.txt (hunk 2 above)
Comment (comzeradd) [inline requirements/prod.txt]: Same here. Celery 4 needs most of these dependencies upgraded, but once we rebase with 3.1.25 these will longer be part of this commit.

**Comment by johngian** [comment]
I am not sure if that really needs a change indeed but have you checked if the task revocation is compatible with v4?

```
> grep -R "revoke"                
remo/voting/admin.py:            celery_app.control.revoke(obj.task_end_id)
remo/voting/admin.py:            celery_app.control.revoke(obj.task_start_id)
remo/voting/models.py:                    celery_app.control.revoke(self.task_end_id)
remo/voting/models.py:                    celery_app.control.revoke(self.task_start_id)
remo/voting/models.py:                    celery_app.control.revoke(self.task_end_id)
remo/voting/models.py:            celery_app.control.revoke(instance.task_start_id)
remo/voting/models.py:            celery_app.control.revoke(instance.task_end_id)
remo/profiles/models.py:        # revoke the tasks in case a user returns sooner
remo/profiles/models.py:            celery_app.control.revoke(rep_profile.unavailability_task_id)
remo/profiles/models.py:            celery_app.control.revoke(mentor_profile.unavailability_task_id)
```

**Comment by comzeradd** [comment]
The `revoke` method remained the same at celery4. But on `remo/voting/admin.py` the import seems that it was wrong.

**Comment by akatsoulas** [comment]
closing in favor of #1351

**Pull request closed without merging**